crates/nu-command/src/formats/from/ssv.rs

```diff
@@ -88,12 +88,12 @@ fn parse_aligned_columns<'a>(
                         let val = match headers.get(i + 1) {
                             Some((_, end)) => {
                                 if *end < l.len() {
-                                    l.get(*start_position..*end)
+                                    l.get(l.char_indices().nth(*start_position).unwrap().0 .. l.char_indices().nth(*end).unwrap().0)
```

Comment:
Instead of using `.unwrap()` can you check to make sure the indices are correct and use `.expect(...)` or return a `ShellError` instead?

Reply:
I've added the `expect()` as you suggested. However, I don't know how to return the ShellError in case of the expect resulting in a panic. I'm not that well versed in Rust. If you could give me a code sample or a code suggestion even that would really help.